src/main/java/com/learning/pages/PageBase.java

```diff
@@ -0,0 +1,12 @@
+package com.learning.pages;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.PageFactory;
+
+public abstract class PageBase {
+  public PageBase(WebDriver driver) {
```

Comment:
You could bind driver to field, this way you won't have to pass it from each BasePage